study.md

```diff
@@ -24,7 +24,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, how Ember "stops breaking the web".
 
 ```md
-<!-- your answer here -->
+What this means is that Ember makes use of the url conventions that are part of the web experience historically, while retaining the benefits of a single page app. There's a link to specific content that a user can bookmark or share, but the user does not need to wait for a page to load.
```

Comment:
Routing is a big part of Ember, and yes the data should be snappy with no waiting for backend servers to response.